src/Propel/Runtime/Map/DatabaseMap.php

```diff
@@ -134,6 +134,35 @@ public function addTableFromMapClass(string $tableMapClass): TableMap
         return $this->getTable($table->getName());
     }
 
+    /**
+     * Dump table maps. Used during configuration generation.
+     *
+     * @return array A dump that can be loaded again with {@link DatabaseMap::loadMapsFromDump()}
```

Comment:
```suggestion
     * @return array<string, mixed> A dump that can be loaded again with {@link DatabaseMap::loadMapsFromDump()}
```
or even more

Reply:
Hey Mark, thank you for the review. Played around with psalm a bit, let me know what you think